lib/standard_api/controller.rb

```diff
@@ -106,7 +106,9 @@ def update
     end
 
     def destroy
-      resources.find(params[:id]).destroy!
+      records = resources.find(params[:id].split(','))
+      ActiveRecord::Base.transaction { records.each(&:destroy!) }
```

Comment:
Shouldn't this be called on the model class? The `ActiveRecord::Base` may not be the same connection that the model is using. See https://guides.rubyonrails.org/active_record_multiple_databases.html